data_steward/cdr_cleaner/cleaning_rules/remove_ehr_data_past_deactivation_date.py

```diff
@@ -41,13 +41,16 @@ def remove_ehr_data_queries(project_id, ticket_number, pids_project_id,
     ehr_union_dataset = bq_utils.get_unioned_dataset_id()
 
     # gets the deactivated participant dataset to ensure it's up-to-date
-    psr.get_deactivated_participants(pids_project_id, pids_dataset_id,
-                                     tablename,
-                                     DEACTIVATED_PARTICIPANTS_COLUMNS)
-
+    df = psr.get_deactivated_participants(pids_project_id, pids_dataset_id,
+                                          tablename,
+                                          DEACTIVATED_PARTICIPANTS_COLUMNS)
+    # To store dataframe in a BQ dataset table
+    destination_table = pids_dataset_id + '.' + tablename
+    psr.store_participant_data(df, project_id, destination_table)
     # creates sandbox and truncate queries to run for deactivated participant data drops
-    queries = rdp.create_queries(project_id, ticket_number, pids_project_id,
-                                 pids_dataset_id, tablename, ehr_union_dataset)
+    queries = rdp.create_queries(project_id, ticket_number, project_id,
```

Comment:
I would probably add a comment here about why we are using `project_id` twice.  Something like,
```
    # the deactivated participants table was stored in the project, project_id, so we are using the same value as
    # the third parameter to rdp to represent the project containing the deactivated_pids table.
```